Cpp_test/lib/ipc-pipe-send.cpp

```diff
@@ -61,7 +74,7 @@ void IPCPipeSend::transfer()
 			}
 			else
 			{
-				throw std::runtime_error(std::string("ERROR: write() while send in pipe"));
+				throw std::runtime_error(std::string("ERROR: write() while send in pipe\n"));
```

Comment:
That's a good idea to add `\n` at the end of the throw message. But you forgot to add it everywhere. Another possibility is to add a "std::endl" in the catch.